Surface_mesh_parameterization/examples/Surface_mesh_parameterization/CMakeLists.txt

```diff
@@ -6,9 +6,6 @@ project( Surface_mesh_parameterization_Examples )
 cmake_minimum_required(VERSION 3.1)
 
 
-# Include this package's headers first
-include_directories (BEFORE . include)
```

Comment:
That should break the build. `./` and `./include/` are not by default.

Reply:
Well.. no, because `./include` does not exist, and `./` does not contain any header.

Reply:
Only if there is such a directory named `include`, and if there is  something that gets included from "."